### ISSUE TITLE
forward-only docs are unclear

### ISSUE THREAD

**Issue by L4Dj29** [OpenSMTPD/OpenSMTPD#867]
Hello,

Thank you for work you do on opensmtpd, you save us from postfix or sendmail.

I am a new user, and learning. local relay and deliver to local mailboxes works and is clear how to do.

I try also to do other simple thing: forward mail for virtual users to some other address.

The docs seem simple, but details are not clear. Adding a more complete example for that would be very helpful, as there are not many blog posts on the topic. For postfix, there are code and docs on this topic. The first part in https://www.cyberciti.biz/faq/linux-unix-bsd-postfix-forward-email-to-another-account/ is an example.

It seems to be a simple example:

```
table domains file:/etc/mail/domains
accept from any for domain <domains> forward-only
accept for any relay
```

Then make a user and add a `.forward` file in `~`. Is there a way without the user or file? The existing doc mentions "or aliases" as alternative, but how? that is not clear.

At first I did not have the last line in the example above, and I found it difficult.. why are emails failing to go to the forward address? I would see the forward address/domain get caught in `rule matched: accept from <anyhost> for  domain domains`, and then lookup the user, but fail and then return `550 Invalid recipient`.

I realized `accept for any relay` might allow that through, but then I get SMTP auth errors with the remote server the message is being forwarded to. When using `forward-only`, it is confusing. Why does it not forward the email? why extra loop to lookup the forwarding address? why is the relay line needed? Or is my example wrong?

I am happy to send PR, but I need to understand how to do it right. Would you agree a better description of the solution would be useful? What is correct way here?

**Comment by poolpOrg** [comment]
Your ticket references the old configuration format.

The configuration format was completely rewritten for the upcoming 6.4.0 release and the man pages were also rewritten almost from scratch to be more clear.

The forward-only delivery method is better documented in the new man page that will be shipped with the 6.4.0 release.

**Comment by wofwofwof** [comment]
I've search a while till I've got this working. This is for the new syntax.

Here are the matching lines in my working config:
table forward-recipients file:/etc/mail/forward-recipients    # contains all forward recipients
action "relay" relay
match from any rcpt-to <forward-recipients> action "relay"

Additionally, I've added this line to my alias file:
local.forward@address.net:        remote.forward@address.net

Then you don't need a .forward file.

cheers wof
